great_expectations/data_context/data_context.py

```diff
@@ -3249,6 +3251,18 @@ def run_checkpoint(
             **kwargs,
         )
 
+    def get_profiler(
+        self,
+        name: Optional[str] = None,
+        ge_cloud_id: Optional[str] = None,
+    ) -> RuleBasedProfiler:
+        return profiler_toolkit.get_profiler(
+            data_context=self,
+            profiler_store=self.profiler_store,
+            name=name,
+            ge_cloud_id=ge_cloud_id,
+        )
+
```

Comment:
@donaldheppner I know we want to move these `toolkit` utilities to an actual class.

Two questions:
* Where do you think that logically falls in our class hierarchy?
* Could that be part of a following PR to refactor the structure of RBP?